lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2868,6 +2875,42 @@ ParseResult FIRStmtParser::parseStaticRefExp(Value &result,
                  parseOptionalExpPostscript(result, false));
 }
 
+/// path ::= 'path(' static_ref ')'
+// NOLINTNEXTLINE(misc-no-recursion)
+ParseResult FIRStmtParser::parsePath(Value &result) {
+  auto startTok = consumeToken(FIRToken::lp_path);
+
+  Value staticRef;
+  if (parseStaticRefExp(staticRef,
+                        "expected static reference expression in 'path'") ||
+      parseToken(FIRToken::r_paren, "expected ')' in 'path'"))
+    return failure();
```

Comment:
This should also parse optional info.

Reply:
I think only statements parse optional info, which propassign already does. e.g.
```firrtl
propassign property, path(wire) @[FooBar.scala 369:27]
```